### ISSUE TITLE
Windows release doesn't include most backends

### ISSUE THREAD

**Issue by timdaman** [mozilla/sccache#362]
Thank you for packing sccache as Windows. I downloaded the last release (0.2.8) and found only the S3 feature present. I use a Redis backend so this is not functional for me. I understand GCS has a dependency issue but there any issue that is preventing the others backends from being enabled in releases?

Here is what I see

I download the following
`
https://github.com/mozilla/sccache/releases/download/0.2.8/sccache-0.2.8-x86_64-pc-windows-msvc.tar.gz`

When I unpack and run it I see
```
C:\Users\vagrant>C:\opt\bin\sccache.exe
sccache: No command specified
sccache 0.2.8

USAGE:
    sccache [FLAGS] [OPTIONS] [cmd]...

FLAGS:
        --dist-auth       authenticate for distributed compilation
    -h, --help            Prints help information
    -s, --show-stats      show cache statistics
        --start-server    start background server
        --stop-server     stop background server
    -V, --version         Prints version information
    -z, --zero-stats      zero statistics counters

OPTIONS:
        --package-toolchain <executable> <out>    package toolchain for distributed compilation
        --stats-format <stats-format>
            set output format of statistics [default: text]  [possible values: text, json]


ARGS:
    <cmd>...

Enabled features:
    S3:        true
    Redis:     false
    Memcached: false
    GCS:       false
    Azure:     false


C:\Users\vagrant>
```

**Comment by timdaman** [comment]
I just checked the release for 0.2.7 and it looks to not have had the issue I identified.
```
C:\Users\vagrant>C:\Users\vagrant\Downloads\sccache-0.2.7-x86_64-pc-windows-msvc\sccache.exe
sccache: No command specified
sccache 0.2.7

USAGE:
    sccache [FLAGS] [OPTIONS] [cmd]...

FLAGS:
    -h, --help            Prints help information
    -s, --show-stats      show cache statistics
        --start-server    start background server
        --stop-server     stop background server
    -V, --version         Prints version information
    -z, --zero-stats      zero statistics counters

OPTIONS:
        --stats-format <stats-format>    set output format of statistics [default: text]  [possible values: text, json]

ARGS:
    <cmd>...

Enabled features:
    S3:        true
    Redis:     true
    Memcached: true
    GCS:       false
    Azure:     true


C:\Users\vagrant>
```

**Comment by Jake-Shadle** [comment]
I've opened #367 to remove both the OpenSSL dependency as well as the `all-windows` feature flag, so hopefully if it is accepted a new version can be released that has all of the features enabled on Windows.

**Comment by luser** [comment]
This should be fixed by #367 once we tag a new release.

**Comment by Jake-Shadle** [comment]
I think #373 also needs to be merged, otherwise this behaviour will persist even with #367.
